genfragments/ThirteenTeV/ADD/createcards_mphotoncp6tune.csh

```diff
@@ -0,0 +1,55 @@
+#!/bin/tcsh
+setenv pwd $PWD
+echo ${pwd}
+
+foreach MD (1000 2000 3000 4000 5000 6000)
+foreach nnn (3 4 5 6 8)
+
+cat>ADDmonoPhoton_MD_${MD}_d_${nnn}_TuneCP5_13TeV_pythia8_cfi.py<<EOF
+
+import FWCore.ParameterSet.Config as cms
+
+
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```

Comment:
could you please change it to concurrent mode one

Reply:
@menglu21 Done